pkg/api/common/helper.go

```diff
@@ -226,3 +226,10 @@ func IsSgxEnabledSKU(vmSize string) bool {
 	}
 	return false
 }
+
+func GenerateUserAssignedIdentityClientIDParameter(isUserAssignedIdentity bool) string {
```

Comment:
Because we're only invoking this func from the `engine` package under `pkg/engine` we don't need to define it in the `common` package as an exportable func.

A good place for this might be `pkg/engine/template_generator.go`.

Reply:
Thanks! Updated.